src/App.js

```diff
@@ -1,58 +1,55 @@
 import React from 'react';
 import styled from 'styled-components';
-import { useStoreActions, useStoreState } from 'easy-peasy';
-import { Button, Box } from 'rebass/styled-components';
-
 import ChatBot from 'react-simple-chatbot';
-import CreatePDF from '~/components/PDF';
+import CreatePDF from './components/PDF';
+import { Box } from 'rebass/styled-components';
 
 export const steps = [
-    {
-      id: '1',
-      message: 'Hallo! Wie kann ich dir helfen?',
-      trigger: '2',
-    },
-    {
-      id: '2',
-      options: [
-        { value: 1, label: 'Meldeformular', trigger: '3' },
-        // { value: 2, label: 'Shared Mobility', trigger: '5' },
-        // { value: 3, label: 'Digitalisation', trigger: '5' },
-      ]
-    },
-    {
-      id: '3',
-      message: 'Wie ist dein Familienname?',
-      trigger: 'name',
-    },
-    {
-      id: 'name',
-      user: true,
-      trigger: '4',
-    },
-    {
-      id: "4",
-      message: "Ihr Formular steht zum Download bereit:",
-      trigger: "5"
-    },
-    {
-      id: "5",
-      component: (<CreatePDF></CreatePDF>),
-      end: true,
-    },
-    // {
-    //   id: "6",
-    //   message: "Okay, let's talk about Shared Mobility!",
-    //   end: true,
-    // },
-    // {
-    //   id: "7",
-    //   message: "Okay, let's talk about Digitalisation!",
-    //   end: true,
-    // },
+  {
+    id: '1',
+    message: 'Hallo! Wie kann ich dir helfen?',
+    trigger: '2',
+  },
+  {
+    id: '2',
+    options: [
+      { value: 1, label: 'Meldeformular', trigger: '3' },
+      // { value: 2, label: 'Shared Mobility', trigger: '5' },
+      // { value: 3, label: 'Digitalisation', trigger: '5' },
+    ]
+  },
+  {
+    id: '3',
+    message: 'Wie ist dein Familienname?',
+    trigger: 'name',
+  },
+  {
+    id: 'name',
+    user: true,
+    trigger: '4',
+  },
+  {
+    id: "4",
+    message: "Ihr Formular steht zum Download bereit:",
+    trigger: "5"
+  },
+  {
+    id: "5",
+    component: (<CreatePDF></CreatePDF>),
+    end: true,
+  },
+  // {
+  //   id: "6",
+  //   message: "Okay, let's talk about Shared Mobility!",
+  //   end: true,
+  // },
+  // {
+  //   id: "7",
+  //   message: "Okay, let's talk about Digitalisation!",
+  //   end: true,
+  // },
 ];
 
-
 const StyledBox = styled(Box)`
```

Comment:
Something that bugs me about styled components is that they tend to pop up everywhere.
I suggest moving them to their own components file instead of having them all around town

Reply:
Sounds good. I would suggest we do this in another step though, this PR does a lot already...